src/pages-and-resources/notes/messages.js

```diff
@@ -0,0 +1,25 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  heading: {
+    id: 'course-authoring.pages-resources.notes.heading',
+    defaultMessage: 'Configure notes',
+  },
+  enable_notes_label: {
+    id: 'course-authoring.pages-resources.notes.enable-notes.label',
+    defaultMessage: 'Notes',
+  },
+  enable_notes_help: {
+    id: 'course-authoring.pages-resources.notes.enable-notes.help',
+    defaultMessage: `Learners can access their notes either in the body of the
+    course of on a notes page. On the notes page, a learner can see all the
+    notes made during the course. The page also contains links to the location
+    of the notes in the course body.`,
+  },
+  enable_notes_link: {
+    id: 'course-authoring.pages-resources.notes.enable-notes.link',
```

Comment:
follow JS convention for declaring key names. i.e in camel case e.g enable_notes_help -> `enableNotesHelp`
everywhere in messages files.